pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java

```diff
@@ -101,11 +100,10 @@ public void updateFunction(final @PathParam("tenant") String tenant,
                                final @FormDataParam("data") InputStream uploadedInputStream,
                                final @FormDataParam("data") FormDataContentDisposition fileDetail,
                                final @FormDataParam("url") String functionPkgUrl,
-                               final @FormDataParam("functionDetails") String functionDetailsJson,
```

Comment:
if you removed this from base, does it break v1 and v2?

Reply:
As I said above, no this is only for v3 API.  The v2 and v1 API still takes in a functionDetails

Reply:
reference: https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Functions.java#L87